index.html

```diff
@@ -21,47 +20,124 @@
       href="https://fonts.googleapis.com/css2?family=Inter+Tight:ital,wght@0,100..900;1,100..900&family=Inter:ital,opsz,wght@0,14..32,100..900;1,14..32,100..900&display=swap"
       rel="stylesheet"
     />
+    <link rel="stylesheet" href="./assets/styles/main.css" />
   </head>
 
   <body class="body">
     <section id="hero"></section>
 
     <section id="about"></section>
 
-    <section id="ammenities" class="p-4">
-      <div class="container container-amenities p-2">
-        <div class="box-title p-2"><h2>Amenities</h2></div>
-        <div class="box-text p-2">
-          <ul class="amenitiesUL">
-            <li>Carport</li>
-            <li>Golf Cart Garages</li>
-            <li>Swimming Pool with Cabana</li>
-            <li>Championship 18-hole Golf Course</li>
-            <li>Tennis Courts</li>
-            <li>Campgrounds</li>
-            <li>Pecan Activity Center</li>
-          </ul>
+    <section id="ammenities" class="p-3">
+      <div
+        class="card text-center pt-3 mx-auto custom-color-card"
+        style="width: 70%"
+      >
+        <div class="card-body">
+          <h2 class="card-title pb-2 custom-color-card-title">Amenities</h2>
+          <div class="card-content p-3">
+            <div class="row pb-3 ps-3">
+              <div class="col">
+                <i class="fa-solid fa-people-roof fa-2xl"></i>
+              </div>
+              <div class="col">
+                <i class="fa-solid fa-car fa-2xl"></i>
```

Comment:
`fa-2xl` is too large for this location
```suggestion
                <i class="fa-solid fa-car fa-xl"></i>
```